ros2cli_common_extensions/package.xml

```diff
@@ -11,6 +11,7 @@
 
   <exec_depend>launch_xml</exec_depend>
   <exec_depend>launch_yaml</exec_depend>
+  <exec_depend>python3-argcomplete</exec_depend>
```

Comment:
why is this needed?
This package doesn't use argcomplete.

If a dependency is missing the argcomplete dependency, it should be added there.

Reply:
Please checkout https://github.com/ros2/ros2_documentation/pull/841#pullrequestreview-463883706

Reply:
I don't think that recommendation was correct, the dependencies have to be added in the packages that are really using it.

If not, what happens if you install `ros2topic` and not `ros2cli_common_extensions`?

Reply:
That was especially meant for people installing `ros-distro-desktop` but yes I agree, that doesn't make sense to skip argcomplete if you only install some command.
Would you mind making this comment there to see how we should fix it?

Reply:
done